### PR TITLE
Requested workshop enrollment receipt changes

### PR DESCRIPTION
**Pull request by joshlory**
Spec: https://docs.google.com/document/d/1qT_VP71lVP8GIf0gFfRPiJjAiEXDwXxiQSpXuPQEX3k/edit

### CODE REVIEW

**Comment by joshlory** [comment]
@mehalshah can you double-check the text changes?  I don't fully understand what gets used where.

**Comment by mehalshah** [comment]
Taking a more detailed look with the spec open

File: dashboard/app/views/pd/workshop_mailer/_supply_list.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 %h3
-  What to Bring
+  What to bring
```
Comment (mehalshah) [inline dashboard/app/views/pd/workshop_mailer/_supply_list.html.haml:2]: Spec has this as a capital B

Comment (mehalshah) [reply dashboard/app/views/pd/workshop_mailer/_supply_list.html.haml:2]: Also there is a section on line 9 on compatible OS and Browsers that needs to be added

Comment (joshlory) [reply dashboard/app/views/pd/workshop_mailer/_supply_list.html.haml:2]: Lowercase "bring" is intentional, to match the casing on the other headers

**Comment by joshlory** [comment]
@mehalshah is there any automated test coverage of these email receipts?

**Comment by mehalshah** [comment]
There's automated testing to make sure that http://localhost-studio.code.org:3000/rails/mailers/pd/workshop_mailer/teacher_enrollment_receipt__csf will render but nothing beyond that

**Comment by joshlory** [comment]
Link to the test?  Thanks!

**Comment by mehalshah** [comment]
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/mailers/previews/traverse_mail_previews_test.rb

**Comment by joshlory** [comment]
This change should have everything except the "Facilitators" added to the logistics section — tracking that as a follow-on PR.

**Pull request merged**